doc/guides/writing-tests.md

```diff
@@ -120,7 +120,7 @@ Avoid timers unless the test is specifically testing timers. There are multiple
 reasons for this. Mainly, they are a source of flakiness. For a thorough
 explanation go [here](https://github.com/nodejs/testing/issues/27).
 
-In the event a test needs a timer, consider using the
+In the event of a ['flaky tests' situation](https://github.com/nodejs/node/wiki/Flaky-tests), consider using the
```

Comment:
Sorry for the late reply, I think the original wording is not too far off, possibly just update to stress only to use timers when necessary:

```
In the event that there is no reasonable alternative to using a timer, consider using the
```